README.md

```diff
@@ -30,8 +30,15 @@ cd phoenicis-javafx/
 mvn exec:java
 ```
 
+For more details on installing on various linux distro's [Install/Build](https://phoenicisorg.github.io/phoenicis/Developers/build/)
```

Comment:
Typo: Should be "distros", not "distro's". Also, "For more details on installing on various linux distros, see:" sounds better.